backend/src/main/validation/PromotionValidation.ts

```diff
@@ -12,6 +14,16 @@ export class PromotionValidation {
     userId: IdValidation.schema,
     placeId: Joi.string().required(),
     discount: DiscountValidation.schema,
+    /**
+     * A promotion must have at least one schedule and can have at most seven (each representing one day of the week)
```

Comment:
Do you mean non-recurring? How come two schedules instead of one?

Reply:
Oooh!! Yes my bad I meant non-recurring 😅  but I was thinking for promotions with a startDate and endDate, we would need exactly X schedules based on X number of days in between start and end? This would be a validation thing though I guess?

I was originally thinking of 2 in the case that it starts Dec 27 and ends Dec 28! But also for the case with at least one schedule- would that be either a promotion that has no end date or the start = end?

Reply:
Not sure if you did this already but maybe we should create an issue of all the validations we can do?

Reply:
Hmm, but startDate and expirationDatecould be months apart and could have a lot of repetition. Note that each schedule doesn't have a calendar date associated with it, only a time and weekday.

If it's Dec 27 and Dec 28, we would have one promotion with that startDate and expirationDate. And then followed by two schedules, one for Sunday and another for Monday, both non-recurring. 

> But also for the case with at least one schedule- would that be either a promotion that has no end date or the start = end?

Do you mean `for the case with exactly one schedule`? That would be a promotion that's only valid on one day.

> Not sure if you did this already but maybe we should create an issue of all the validations we can do?

Do you mean for the frontend or backend?

Reply:
Ahh that's true and I see that makes sense! Probably on FE since we should do all our validations before storing it in the db and whatnot

Reply:
Yup I agree!